.github/workflows/abtesting.yml

```diff
@@ -17,6 +17,9 @@ on:
 
 jobs:
   pod-lib-lint:
+    # Don't run on private repo unless it is a PR.
```

Comment:
Also need to delete the on: push: above - here and other .ymls that have it.

Reply:
Do you mean we would like to get rid of post-submit checks even in public repo?

Reply:
Yes, they're inconsistently used and not a lot of value.

Reply:
done